client/viame-web-common/components/TrackDetailsPanel.vue

```diff
@@ -0,0 +1,394 @@
+        <v-row
+          no-gutters
+          class="align-center"
+        >
+          <b>Confidence Pairs:</b>
+          <v-spacer />
+          <v-tooltip
+            open-delay="200"
+            bottom
+            max-width="200"
+          >
+            <template #activator="{ on }">
+              <v-btn
+                disabled
+                outlined
+                x-small
+                class="my-1"
+                v-on="on"
+                @click="confidencePairsAdd"
+              >
+                <v-icon small>
+                  mdi-plus
+                </v-icon>
+                Pair
+              </v-btn>
+            </template>
+            <span>Add a new Confidence Pair</span>
+          </v-tooltip>
+        </v-row>
+      </div>
+
+      <v-row
+        dense
+        class="scroll-section confidence shrink"
+      >
+        <v-col dense>
+          <v-row
+            v-for="(pair, index) in selectedTrack.confidencePairs"
+            :key="index"
+            class="ml-1"
+            dense
+            style="font-size: 0.8em"
+          >
+            <v-col cols="1">
+              <div
+                class="type-color-box"
+                :style="{
+                  backgroundColor: typeStylingRef.color(pair[0]),
+                }"
+              />
+            </v-col>
+            <v-col>
+              {{ pair[0] }}
+            </v-col>
+            <v-col>
+              {{ pair[1].toFixed(2) }}
+            </v-col>
+          </v-row>
+        </v-col>
+      </v-row>
+      <attribute-subsection
+        mode="Track"
+        :attributes="attributes"
+        :edit-individual="editIndividual"
+        @edit-attribute="editAttribute($event)"
+        @set-edit-individual="setEditIndividual($event)"
+        @add-attribute="addAttribute"
+      />
+      <attribute-subsection
+        mode="Detection"
+        :attributes="attributes"
+        :edit-individual="editIndividual"
+        @edit-attribute="editAttribute($event)"
+        @set-edit-individual="setEditIndividual($event)"
+        @add-attribute="addAttribute"
+      />
+    </template>
+    <v-dialog
+      :value="editingAttribute != null"
+      max-width="350"
+    >
+      <attribute-editor
+        v-if="editingAttribute != null"
+        :selected-attribute="editingAttribute"
+        :error="editingError"
+        @close="closeEditor"
+        @save="saveAttribtueHandler"
+        @delete="deleteAttributeHandler"
+      />
+    </v-dialog>
+  </v-card>
+</template>
+
+<style lang="scss" scoped>
+.attribute-item-value {
+  max-width: 80%;
+  margin: 0px;
+  &:hover {
+    cursor: pointer;
+    font-weight: bold;
+  }
+}
+.attribute-name {
+  font-size: 0.8em;
+  max-width: 50%;
+  min-width: 50%;
+}
+.border-highlight {
+  border-top: 1px solid gray;
+  border-bottom: 1px solid gray;
+  color: white;
+  font-weight: bold;
+  font-size: 0.9em;
+  padding: 4px 10px;
+  background-color: #272727;
+}
```

Comment:
Maybe refactor this to a `BorderHighlight.vue` with a content slot.

Can probably be a pure functional component.

Reply:
I added in a `PanelSubsection.vue` component for structuring the data elements that need to be placed in the Panel and have a scrolling subsection as well.  Used for the ConfidenceEditor.vue and both AttributesSubsection.vue components.

I didn't want to go too far into it, but in the future we could add a `settings` slot and perhaps replace the TypeList and TrackList so that everything has a standard style with a header, expandable settings section, and then a scroll area.  It should help with all these vertical subsections that can independently scrolled.